scapy/layers/tls/crypto/cipher_block.py

```diff
@@ -44,14 +44,14 @@ class _BlockCipher(six.with_metaclass(_BlockCipherMetaclass, object)):
 
     def __init__(self, key=None, iv=None):
         self.ready = {"key": True, "iv": True}
-        if key is None:
+        if not key:
```

Comment:
Is this needed? Why? Replacing `if X is None` by `if not X` or `if X is not None` by `if X` should be done only if necessary.

Same questions for other similar replacements.

Reply:
You’re right. I used regex to replace all of those at once. Will rewind those changes tonight